commit/merkleroot/outcome.go

```diff
@@ -71,12 +72,16 @@ func reportRangesOutcome(
 	lggr logger.Logger,
 	consensusObservation ConsensusObservation,
 	maxMerkleTreeSize uint64,
+	dstChain cciptypes.ChainSelector,
 ) Outcome {
 	rangesToReport := make([]plugintypes.ChainRange, 0)
 
 	observedOnRampMaxSeqNumsMap := consensusObservation.OnRampMaxSeqNums
 	observedOffRampNextSeqNumsMap := consensusObservation.OffRampNextSeqNums
+	observedRMNRemoteConfig := consensusObservation.RMNRemoteConfig
+
 	offRampNextSeqNums := make([]plugintypes.SeqNumChain, 0)
+	rmnRemoteConfig := rmntypes.RemoteConfig{}
```

Comment:
This looks pretty misplaced over here, lets move it closer to where its assigned.

Reply:
Prefer `var rmnRemoteConfig rmntypes.RemoteConfig` for declarations.

Reply:
The idea was to have it declare where all the returned values where. But I got your point, moving it closer to where it's being used